app/assets/stylesheets/welcome.scss

```diff
@@ -1,3 +1,38 @@
-// Place all the styles related to the welcome controller here.
-// They will automatically be included in application.css.
-// You can use Sass (SCSS) here: http://sass-lang.com/
+
+h1 {
+  font-size: 70px;
+}
+
+h2 {
+ color: #dc6312;
+}
+
+.welcome-header {
+  padding: 100px;
+}
+body {
+  font-family: 'Lobster', cursive;
+  background: rgba(21, 193, 220, 0.4);
+}
+
+.inner-well {
+  background: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.